cmd/ebrelayer/txs/relayToCosmos.go

```diff
@@ -71,3 +70,36 @@ func RelayToCosmos(factory tx.Factory, claims []*types.EthBridgeClaim, cliCtx cl
 
 	return nil
 }
+
+// SignProphecyToCosmos broadcast the sign prophecy message to cosmos
+func SignProphecyToCosmos(factory tx.Factory, signProphecy ethbridge.MsgSignProphecy, cliCtx client.Context, sugaredLogger *zap.SugaredLogger) {
+	var messages []sdk.Msg
+
+	messages = append(messages, &signProphecy)
+
+	sugaredLogger.Infow("RelayToCosmos building, signing, and broadcasting", "messages", messages)
+	// TODO this WithGas isn't correct
+	// TODO we need to investigate retries
+	// TODO we need to investigate what happens when the transaction has already been completed
+	err := tx.BroadcastTx(
+		cliCtx,
+		factory.
```

Comment:
We'll just accept the changes of how this is done on the current develop branch as they have the necessary changes for 0.42.